website/client/components/groups/group.vue

```diff
@@ -397,6 +397,10 @@ export default {
         description: true,
         challenges: true,
       },
+	  chat: {
+		SubmitDisable: false,
```

Comment:
could you use the `camelCase` naming for the inner properties here?

Reply:
My mistake! Missed this when I refactored. I'll get that fixed.